### ISSUE TITLE
accept entry without affiliation

### ISSUE THREAD

**Issue by jcolomb** [marton-balazs-kovacs/tenzing#60]
sometimes only looking at contribution statement, so not needing the affiliation. 
Change from error to warnings when there is no affiliation

**Comment by marton-balazs-kovacs** [comment]
You are right! This is a big topic and we should definitely review the validation function. Changes have to be made on several fronts regarding the validation. This is one of them. Another would be being more permissive with the column names. If we want to make affiliations not necessary we have to include a display logic so the outputs that require this info are disabled. I plan to functionalize the display logic part better so it would be easy to change and upgrade them. Let's make this a high priority for the next release.

**Comment by alexholcombe** [comment]
In conversation today, Marton and I agreed to have the app "degrade gracefullly" with degenerate input, in that it would still provide outputs (as even partial outputs are useful for some use cases, so it can be somewhat irritating to users to refuse to provide an output) in almost all cases but prominently indicate warnings, and indicate missing information with NA or NULL in all-caps, written in red (except for raw-text output which doesn't allow color).
